R/snapshots.R

```diff
@@ -0,0 +1,116 @@
+#' Run Simulations From Snapshot Files
+#'
+#' @param ... character strings, path to snapshot files or a directory containing snapshot files 
+#' @param output character string, path to the output directory where to write simulation results
+#' @param csv logical, whether to export the results as csv (default = TRUE)
+#' @param pkml logical, whether to export the results as pkml (default = FALSE)
+#' @param xml logical, whether to export the results as xml (default = FALSE)
```

Comment:
would rename the arguments to _**export**XXX_ for clarity (e.g. `exportCsv `etc.)